src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -179,6 +181,29 @@ def sort_code_mapping_configs(
                     sorted_configs.insert(0, config)
         return sorted_configs
 
+    def get_commit_sha(
+        self,
+        integration: Integration,
+        organization_id: str,
+        line_no: int,
+        filepath: str,
+        current_config: JSONData,
+    ) -> Optional[str]:
+        githubIntegration = GitHubIntegration(integration, organization_id)
+        try:
+            git_blame_list = githubIntegration.get_blame_for_file(
+                current_config["repository"],
+                filepath,
+                current_config["config"]["defaultBranch"],
+                line_no,
+            )
+            for blame in git_blame_list:
+                if line_no >= blame["startingLine"] and line_no <= blame["endingLine"]:
+                    return str(blame["commit"]["oid"])
+        except Exception:
+            logger.exception("Could not get git blame")
+        return None
```

Comment:
```suggestion
        return commitSha
```
I prefer having a single return at the bottom if at all possible. It's a better pattern to follow as the function becomes larger.